schedule.yaml

```diff
@@ -71,4 +71,4 @@ static_knockout:
         start_time: 2019-08-16 14:03:00 +1
         # Ends at 14:06:30
         #         S2,    S3,    S1,    S4
-        teams: ['000', '001', '010', '011']
+        teams: ['000', '010', '001', '011']
```

Comment:
Please could you also update the preceding line which has the apparent seedings on it.